includes/Vendor/SetupWizard.php

```diff
@@ -123,7 +123,7 @@ public function setup_wizard_header() {
         </head>
         <body class="wc-setup wp-core-ui">
             <?php if ( ! empty( $this->custom_logo ) ) { ?>
-                <h1 id="wc-logo"><a href="<?php echo esc_url( home_url() ) ?>"><img src="<?php echo esc_url( $this->custom_logo ); ?>" alt="<?php echo esc_attr( get_bloginfo( 'name' ) ); ?>" /></a></h1>
+                <h1 id="wc-logo"><a href="<?php echo esc_url( home_url() ) ?>"><img src="<?php echo esc_url( $this->custom_logo ); ?>" alt="<?php echo esc_attr( get_bloginfo( 'name' ) ); ?>" style="width : 100%;" /></a></h1>
```

Comment:
This is wrong thinking, if you use 100% width then it will be problem when use big image, also we don't allow inline CSS, try to use on source files for css/js

Reply:
do you want me to remove this line and add this to css?

Reply:
Right, please remove the inline css and use the css codes on this file `assets\css\setup.css` with `CSS @media Rules` because you want to use for only mobile devices